### PR TITLE
ADD Compatibilty to Postgresql 15 on migrations

### PR DESCRIPTION
**Pull request by arpagon**
ADD compatibility to Supabase PostgreSQL 15

https://github.com/supabase/supabase/discussions/1501#discussioncomment-715670

### CODE REVIEW

File: web/supabase/config.toml [hunk 1]
```diff
@@ -19,7 +19,8 @@ max_rows = 1000
 port = 54322
 # The database major version to use. This has to be the same as your remote database's. Run `SHOW
 # server_version;` on the remote database to check.
-major_version = 14
+shadow_port = 54320
```
Comment (chitalian) [inline web/supabase/config.toml:22]: Can we not change the default shadow port?

File: web/supabase/migrations/20221201092605_response_request_view.sql [hunk 2]
```diff
@@ -23,5 +45,5 @@ GRANT ALL ON TABLE public.response_and_request TO postgres;
 GRANT ALL ON TABLE public.response_and_request TO anon;
 GRANT ALL ON TABLE public.response_and_request TO service_role;
 
--- THANK YOU REDDIT https://www.reddit.com/r/Supabase/comments/txq9o9/rls_views_and_functions/
-ALTER VIEW response_and_request OWNER TO authenticated;
\ No newline at end of file
+-- -- THANK YOU REDDIT https://www.reddit.com/r/Supabase/comments/txq9o9/rls_views_and_functions/
+-- ALTER VIEW response_and_request OWNER TO authenticated;
```
Comment (chitalian) [inline web/supabase/migrations/20221201092605_response_request_view.sql:49]: ahh I know why you did this, but we shouldn't commit this. I will make an issue to solve this

**Comment by chitalian** [comment]
The think the issue you were facing is because of
`ALTER TABLE public.user_metrics OWNER TO authenticated;` and not a PG 15 vs 14 issue. I am going to follow up with a PR to fix this issue by changing the auth on these views as a follow up instructions... This is an issue with Supabase

File: web/supabase/config.toml (hunk 1 above)
Comment (arpagon) [reply web/supabase/config.toml:22]: Actually this is the default shadow port. 

https://supabase.com/docs/reference/cli/config

**Comment by chitalian** [comment]
Merging for now, will do a fast follow on SQL changes,

**Comment by chitalian** [comment]
@arpagon if you wanna take a look at the migrations now, we cleaned it up a bit. 

Thanks for your PR!

Here is it cleaned up a bit.
https://github.com/Helicone/helicone/pull/142

Luckily we got rid of any views that relied on RLS, which is why we had to change the owner of the views to authenticated.

File: web/supabase/config.toml (hunk 1 above)
Comment (chitalian) [reply web/supabase/config.toml:22]: I ended up removing this, because it was failing to push with this on with my testing. Not sure why

**Comment by arpagon** [comment]
@chitalian  Just out of curiosity, did you update to PG15 in the main backend in Supabase?

**Pull request merged**